packages/abi-coder/src/encoding/coders/StructCoder.test.ts

```diff
@@ -13,8 +13,7 @@ import { StructCoder } from './StructCoder';
  * @group browser
  */
 describe('StructCoder', () => {
-  const STRUCT_NAME = 'TestStruct';
-  const coder = new StructCoder(STRUCT_NAME, {
+  const coder = new StructCoder('std::vec::Vec', {
```

Comment:
A vector can't have `a` and `b` properties. I think it's okay to set it as `struct TestStruct`.

Reply:
0d0620d3151eaa0fb192a4b66ac47880ac343636